aws/resource_aws_ecs_service.go

```diff
@@ -195,6 +198,19 @@ func resourceAwsEcsService() *schema.Resource {
 	}
 }
 
+func resourceAwsEcsServiceImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	if len(strings.Split(d.Id(), "/")) != 2 {
+		return []*schema.ResourceData{}, fmt.Errorf("[ERR] Wrong format of resource: %s. Please follow 'cluster-name/service-name'", d.Id())
+	}
+	cluster := strings.Split(d.Id(), "/")[0]
+	name := strings.Split(d.Id(), "/")[1]
+	log.Printf("[DEBUG] Importing ECS service %s from cluster %s", name, cluster)
+
+	d.SetId(name)
+	d.Set("cluster", cluster)
```

Comment:
Do you mind setting both ID and `cluster` to ARNs here to keep expectations aligned with what's documented?

Thanks.